CONTRIBUTING.md

```diff
@@ -38,17 +46,89 @@ OS            | Supported tests
 OS X          | Android, iOS
 Windows       | Android
 
+### Test descriptions
+
+The tests first build the app.
+
+They then check if the required emulators are currently running.
+
+If an Android emulator is not running, it attempts to boot an Android emulator named ```emulator```. You can specify an emulator by adding ```--androidemu yourEmulatorNameHere``` as a command line option to the gulp task.
+
+If an iOS simulator is not running, it attempts to boot the latest iOS iPhone simulator. You can specify a simulator by adding ```--iosemu yourSimulatorNameHere``` as a command line option to the gulp task.
```

Comment:
Given that the iOS platform uses simulators and not emulators, would it be a good idea to rename this option to something like `--iossim`? If we do rename it, we would lose consistency with the android option. If consistency is a concern, we can pick a name that can be used for both, like `--target`